src/ontology/fbbt-edit.obo

```diff
@@ -118272,6 +118272,41 @@ intersection_of: part_of FBbt:00058008 ! larval anterior-middle midgut junction
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-04-14T13:26:51Z
 
+[Term]
+id: FBbt:00058013
+name: TE neuron
+def: "An extrinsic neuron of the optic lobe that is only present during pupal development. The projections of these neurons wrap the dorsal and ventral edges of all optic lobe neuropils except the lamina. They originate from progenitors at the tips of the outer proliferation centre and are cleared by apoptosis during late pupal stages. They specifically express dimmed (FBgn0023091) and portabella (FBgn0043005) (Ozel et al., 2021)." [FlyBase:FBrf0247762]
+subset: cur
+synonym: "transient extrinsic neuron" EXACT []
+is_a: FBbt:00005125 ! interneuron
+relationship: expresses http://flybase.org/reports/FBgn0023091 ! dimm
+relationship: expresses http://flybase.org/reports/FBgn0043005
+relationship: part_of FBbt:00001930 ! embryonic/larval optic lobe
```

Comment:
Given that this is an 'extrinsic' neuron, it is probably not 'part of' the optic lobe, use 'overlaps', 'has soma location', 'has synaptic IO in' as appropriate.
depending on how early/late in the pupa they arise you might want an adult term (if they are fairly early you are probably correct to use larval term).